src/services/rawReportingExportService.ts

```diff
@@ -0,0 +1,43 @@
+import RawReportingEventsRepository from "src/repositories/raweventsRepository";
+import { sendReporting } from "../services/sendEmailService";

```

Comment:
services should not be interdependent (apart from types, and there might be exceptions but this doesn't seem like one) - so as mentioned in the controller file the sendReporting function should be used in controller